docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -393,6 +393,12 @@ properties:
   - Fully qualified name of the Java class to use for obtaining AWS credentials.
     Can be used to supply a custom credentials provider.
   -
+* - `hive.metastore.glue.use-web-identity-token-credentials-provider`
+  - If you are running Trino on Amazon EKS, and authenticate using a Kubernetes
+    service account, you can set this property to `true`. Setting to `true` forces
+    Trino to not try using different credential providers from the default credential
+    provider chain, and instead directly use credentials from the service account.
+  -
```

Comment:
We should document the default value here.

Reply:
Added